src/components/common.js

```diff
@@ -508,7 +508,12 @@ export const Switch = forwardRefWithName('Switch', ({ onChange, onLabel = 'True'
 })
 
 export const HeroWrapper = ({ showMenu = true, bigSubhead = false, children }) => {
-  const heavyWrapper = text => bigSubhead ? b({ style: { whiteSpace: 'nowrap' } }, [text]) : text
+  const heavyWrapper = text => bigSubhead ? strong({ style: { whiteSpace: 'nowrap' } }, [text]) : text
```

Comment:
For improving accessibility as [described by @cahrens](https://github.com/DataBiosphere/terra-ui/pull/3134#discussion_r895971066)